centos/7/Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN yum -y install \
     wget \
     curl \
     pygpgme \
-    yum-utils
+    yum-utils \
+    perl-IPC-Cmd
```

Comment:
I don't understand what is the problem and how this perl package related to it.

Can you, please, add some explanation right into the docker file?